### ISSUE TITLE
osd.deploy threw an exception. Exception: Mine on data5.ceph for cephdisks.list

### ISSUE THREAD

**Issue by tserong** [SUSE/DeepSea#1291]
### Description of Issue/Question

The error message "Mine on $(hostname) for cephdisks.list" is remarkably unhelpful.

### Setup
deepsea-0.8.5 on SLES 12 SP3 with salt 2016.11.4

### Steps to Reproduce Issue
Somehow, when running stage 3, the mines weren't up to date. I don't recall seeing this before, and `salt '*' mine.update` fixed it, but the error message displayed isn't a lot to go on. Something like "mine on $hostname not up to date, try `salt '*' mine.update`" would be nice :-)

**Comment by jschmid1** [comment]
Agreed. Here's the line#  https://github.com/SUSE/DeepSea/blob/SES5/srv/salt/_modules/osd.py#L644

**Comment by marcosps** [comment]
Is there a way to reproduce this issue? Also, is this behavior reproduced in newer versions of DeepSea?

**Comment by tserong** [comment]
I honestly have no idea how I originally ended up with the mines out of date when running stage 3, but you might try `salt '*' mine.delete 'cephdisks.list'`, or even `salt '*' mine.flush` before running stage 3, to see if you hit the issue then (based on https://docs.saltstack.com/en/latest/ref/modules/all/salt.modules.mine.html, I haven't tested this myself)
